### DISCUSSION TITLE
Using GoReleaser for non-go projects

### DISCUSSION THREAD

**Discussion by abemedia** [goreleaser/goreleaser#1470] [Ideas, Issue Triage, and General discussions]
I've been using GoReleaser for Go projects for a while and love it, however a lot of the features would also be awesome for non-go projects.

I have a use-case where I have a JavaScript project where I need the result of the build to be put in an archive on Gitlab releases with a changelog, the same way GoReleaser does so I was thinking would it be possible to achieve this? Just bundling the static files, creating the release and changelog...

I realise I can just use tar & curl from my CI pipeline to create and upload the archive and there's other tools for generating the changelog but the beauty of GoReleaser is that you have a single, portable config for creating releases, changelogs, dockerfiles etc. and I think many other projects could benefit from this.

**Comment by caarlos0** [comment]
Hey, no there is not.

I think that for javascript projects there is a `semantic-release` tool that does everything...
